lang/en/texts/mobile-app-open-food-facts.html

```diff
@@ -57,4 +59,29 @@ <h1>Install the Open Food Facts mobile app</h1>
 </scripts>
 
 <initjs>
-</initjs>
\ No newline at end of file
+</initjs>
+
+<br />
+
+<div class="panel callout">
+<h2>It's new !</h2>
+
+<p>
+<b>July 2019 :</b> <a href="https://www.helloasso.com/associations/open-food-facts/collectes/une-meilleure-alimentation-pour-une-meilleure-sante-on-se-mobilise">A better diet for better health: let's mobilize!</a><br>Crowdfunding campaign to imagine and create together the ideal app!</p>
```

Comment:
This "helloasso" thing doesn't seem to be localized very well. If we need to link to a French site, we should at least add 
`hreflang` and tell users they may be sent to a site they won't understand.
```suggestion
<b>July 2019:</b> <a href="https://www.helloasso.com/associations/open-food-facts/collectes/une-meilleure-alimentation-pour-une-meilleure-sante-on-se-mobilise" hreflang="fr">A better diet for better health: let's mobilize! (French)</a><br>Crowdfunding campaign to imagine and create together the ideal app!</p>
```